doc/source/missing_data.rst

```diff
@@ -759,12 +759,7 @@ the ``dtype="Int64"``.
 
 .. ipython:: python
 
-   s = pd.Series(np.random.randn(5), index=[0, 2, 4, 6, 7],
-                 dtype="Int64")
-   s > 0
-   (s > 0).dtype
-   crit = (s > 0).reindex(list(range(8)))
-   crit
-   crit.dtype
```

Comment:
@TomAugspurger was there something specific you wanted to show with the above code? It seems you copied this from above about booleans becoming object, but which doesn't seem relevant for integer NA

(the actual fix here is the first line, you can't convert floats from random.randn to Integers)

Reply:
I don't recall anything specific goal behind that example.